zkstack_cli/README.md

```diff
@@ -30,6 +30,16 @@ zkstackup  --local
 
 This command installs `zkstack` from the current repository.
 
+### Add Autocompletion
+
+TODO: finalize section
+
+```bash
+zkstack --generate=zsh > /usr/local/share/zsh/site-functions/_zkstack
+compinit
+zkstack --<TAB>
+```
+
```

Comment:
Seems not needed anymore

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/3097/commits/039e5dd7c36b4f56cd02f6f0509217db4fbc73eb